src/modules/simulator/simulator_mavlink.cpp

```diff
@@ -575,7 +575,7 @@ void Simulator::pollForMAVLinkMessages(bool publish)
 				mavlink_status_t status;
```

Comment:
The use of status is wrong here: It *has to be* a class member and be initialised with `mavlink_reset_channel_status(uint8_t chan);`.

Commander takeoff might be overridden by manual control. We can have a look at that together.